packages/structure/src/model/RWRoute.ts

```diff
@@ -147,8 +147,9 @@ export class RWRoute extends BaseNode {
     return this.prerender
   }
 
+  // TODO: this is just a placeholder / reminder
```

Comment:
How come you moved the TODO out of the function?

Reply:
Oh, nothing seriously important - just to keep it with the `// eslint-disable-next-line` comment. There's a common practice of putting explanations alongside disables that sometimes gets reinforced with [`@eslint-community/eslint-comments/require-description`](https://eslint-community.github.io/eslint-plugin-eslint-comments/rules/require-description.html).

Reply:
I got rid of the entire method in another PR, so now we don't have to worry about this anymore :D 

https://github.com/redwoodjs/redwood/pull/10923